src/components/Book.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const Book = (props) => {
+  // eslint-disable-next-line react/prop-types
```

Comment:
- [ ] Let's try to not comment linters as they help us write cleaner code.

Reply:
Hi @Ghiftee 
Thanks for the heads up, I've fixed the issue.